shanoir-ng-front/src/app/processing/execution/execution.component.ts

```diff
@@ -134,182 +137,204 @@ export class ExecutionComponent implements OnInit {
     }
 
     // Here we create a bunch of executions with default parameters
+
     onApplyParameters() {
         this.parametersApplied = false;
 
         let availableDatasets: Dataset[] = Array.from(this.selectedDatasets);
 
+        this.datasetsOptions = [];
+        availableDatasets.forEach(dataset => {
+            this.datasetsOptions.push(new Option<Dataset>(dataset, dataset.name + '(' + dataset.id + ')'));
+        });
+
         // By default, we order by alphabtical order
         // TODO: Propose another possible order (by ID?)
         availableDatasets.sort((a: Dataset, b: Dataset) => {
             return a.name.localeCompare(b.name);
         })
 
-        let execution: Execution = new Execution();
-
-        execution.name = this.executionForm.get("execution_name").value;
-        execution.pipelineIdentifier = this.pipeline.identifier;
-        execution.timeout = 20;
-        execution.inputValues = {};
-
         this.pipeline.parameters.forEach(
             parameter => {
-                if (parameter.type == ParameterType.File) {
+                if (this.isAFile(parameter)) {
                     // If we have a file, we try to set up the adapted dataset
                     // We try to find all adapted datasets
-                    let datasetFilter: RegExp = new RegExp(this.executionForm.get(parameter.name).value);
+                    let value = this.executionForm.get(parameter.name).value ? this.executionForm.get(parameter.name).value  : "";
```

Comment:
Maybe we should initialize all file parameters with '*' (if it works ? I don't remember ?)

Reply:
Add an infobox for user ("filtering expression blabla")